docs/user-roles.rst

```diff
@@ -0,0 +1,39 @@
+.. _user-roles:
+
+==========
+User Roles
+==========
+
+An overview of user roles and their privileges for organizations and projects.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. _org-roles:
+
+Organization roles
+==================
+
+An *Organization Admin* can add users to and remove users from an organization.
+Admins can perform all available operations for projects and services. They have access to the organization's Audit Log.
```

Comment:
this is the start of a new para, so it should be separated from the previous para with an empty newline character